public/sass/modules/mixins/_skewed_div.scss

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------*/
+/*-- Slanted Background PseudoEl --*/
+/*---------------------------------*/
+
+/* - may be needed - */
+
+div .skewed-container .inner-div {
```

Comment:
I would keep these things separate. Presentation away from functional Sass. 

Also, I make it a habit to always indent selectors.